src/TypeSpec.Extension/Emitter.Csharp/src/type/operationFinalStateVia.ts

```diff
@@ -1,8 +1,27 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
+import { FinalStateValue } from "@azure-tools/typespec-azure-core";
 
 export enum OperationFinalStateVia {
     AzureAsyncOperation,
     Location,
-    OriginalUri
+    OriginalUri,
+    OperationLocation
+}
+
+export function convertLroFinalStateVia(
```

Comment:
Please add unit test for this emitter change.

Reply:
added